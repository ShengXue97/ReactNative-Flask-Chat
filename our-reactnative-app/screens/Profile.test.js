import React from 'react';
import Profile from './Profile.js';


import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<Profile />).toJSON();
  expect(tree).toMatchSnapshot();
});
