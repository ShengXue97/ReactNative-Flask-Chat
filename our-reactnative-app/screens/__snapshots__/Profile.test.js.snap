// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "#376DCF",
          "borderBottomColor": "#dddddd",
          "borderBottomWidth": 1,
          "height": 50,
        }
      }
    >
      <View>
        <TextInput
          allowFontScaling={true}
          placeholder="Timetable"
          placeholderTextColor="white"
          rejectResponderTermination={true}
          style={
            Object {
              "flex": 1,
              "fontWeight": "700",
              "paddingLeft": 15,
              "paddingTop": 20,
            }
          }
          underlineColorAndroid="transparent"
        />
      </View>
    </View>
  </View>
</RCTSafeAreaView>
`;
